org-brain.el

```diff
@@ -3025,11 +3038,22 @@ LINK-TYPE will be \"brain\" by default."
                         (org-brain-title entry))
                 nil choice)
              (org-brain-add-resource
-              (concat "file:" (file-relative-name
+              (concatenate 'string
+			   "file:"
+			   (file-relative-name
                                (buffer-file-name)
-                               (file-name-directory (org-brain-entry-path choice))))
-              (concat (and (stringp org-brain-backlink) org-brain-backlink)
-                      (file-name-base))
+                               (file-name-directory (org-brain-entry-path choice)))
+			   (if (and org-brain-backlink-heading
+				     (ignore-errors (org-get-outline-path t)))
```

Comment:
You could use an `if-let` around `(org-brain-outline-path t)`, but its a minor thing and I can change it after the merge. Also I would prefer spaces instead of tabs, but I can change that too.

Reply:
Updated this. Let me know if it works like that. Thanks for the tip.